RxSwift/Observables/Catch.swift

```diff
@@ -169,7 +169,7 @@ final private class Catch<Element>: Producer<Element> {
 // catch enumerable
 
 final private class CatchSequenceSink<S: Sequence, O: ObserverType>
-   : TailRecursiveSink<S, O>
+    : TailRecursiveSink<S, O>
```

Comment:
Just FYI this was misaligned before my PR (with a few others), so I've aligned those back.